app/api/helpers/payment.py

```diff
@@ -243,3 +244,26 @@ def charge_source(order_identifier):
                  source=order.order_notes,
                  )
         return charge
+
+
+class OmisePaymentsManager(object):
+    """
+    Class to manage Omise Payments
+    """
+
+    @staticmethod
+    def charge_payment(order_identifier, token):
+        if get_settings()['app_environment'] == Environment.PRODUCTION:
+            omise.api_secret = get_settings()['omise_live_secret']
+        else:
+            omise.api_secret = get_settings()['omise_test_secret']
+        order = safe_query(db, Order, 'identifier', order_identifier, 'identifier')
+        charge = omise.Charge.create(
+                amount=order.amount,
```

Comment:
Omise docs needs the amount to be in integer so we should make it
`amount=int(order.amount)`

Reply:
Yeah, sure